jsx/Form.js

```diff
@@ -40,23 +40,6 @@ class FormElement extends Component {
     const colSize = Math.floor(maxColumnSize / columns);
     const colClass = 'col-xs-12 col-sm-' + colSize + ' col-md-' + colSize;
 
-    // Render elements from JSON
-    const filter = this.props.formElements;
-
-    Object.keys(filter).forEach(function(objKey, index) {
-      const userInput = this.props.onUserInput ? this.props.onUserInput : filter[objKey].onUserInput;
-      const value = filter[objKey].value ? filter[objKey].value : '';
-      formElementsHTML.push(
-        <div key={'el_' + index} className={colClass}>
-          <LorisElement
-            element={filter[objKey]}
-            onUserInput={userInput}
-            value={value}
-          />
-        </div>
-      );
-    }.bind(this));
-
```

Comment:
kept in **24.1**

_Unclear why this was removed. Better to keep it as to not cause issues when joining main._

to be tested, although basic assumption is to adopt **24.1**